Instructions.md

```diff
@@ -0,0 +1,32 @@
+# Instructions for testing RBAC support for Linux in .NET 5 preview 6
+
+1. Install .NET 5 preview 6 SDK
+   1. Follow the instructions at https://github.com/dotnet/installer#installers-and-binaries. Key steps are:
+   2. Download the sdk tar.gz at https://aka.ms/dotnet/net5/preview6/Sdk/dotnet-sdk-linux-x64.tar.gz and extract to a local directory. Add this directory to your PATH.
+   3. Add dotnet5 feed to your NuGet.config. You may need to add a NuGet.config file to your project directory if it doesn't exist. For example:
+```xml
+<configuration>
+  <packageSources>
+    <add key="dotnet5" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
+2. Modify your project
+   1. Update the TFM of your project to `net5.0`
+   2. Add the following packages to your project
+```xml
+<PackageReference Include="System.DirectoryServices.Protocols" Version="5.0.0-preview.6.20305.6" />
+<PackageReference Include="Microsoft.AspNetCore.Authentication.Negotiate" Version="5.0.0-preview.6.20305.3" />
+```
+   3. Add the file [LinuxAdapter.cs](https://github.com/JunTaoLuo/LinuxRBAC/blob/master/LinuxAdapter.cs) to your project
+   4. Modify the `ConfigureServices` method to the following:
+```C#
+services.AddAuthentication(NegotiateDefaults.AuthenticationScheme)
+    .AddNegotiate(options => options.Events = new NegotiateEvents()
+    {
+        OnAuthenticated = context => LinuxAdapter.OnAuthenticated(context, "user@DOMAIN.net", "<Password>")
```

Comment:
I ran into a bit of trouble with using default credentials but I aim to update the API to looks something like 
```c#
LinuxAdapter.OnAuthenticated(context, "DOMAIN.net")
```
When I get that sorted out.